### PR TITLE
Don't allow to export licenses

### PR DESCRIPTION
**Pull request by uilianries**
Both exports and exports_sources should not provide a LICENSE file. It should be collected from the downloaded files.

closes https://github.com/conan-io/hooks/issues/85
Wiki:

#### **<a name="KB-H023">#KB-H023</a>: "EXPORT LICENSE"**

Recipe package should not contain a license file, as all recipes are under the same license type.

```python
class SomeRecipe(ConanFile):
    ...

    exports = "LICENSE.md" # not allowed
```

There is a complete explanation on [FAQ](https://github.com/conan-io/conan-center-index/wiki/FAQ#should-recipes-export-a-recipes-license).

### CODE REVIEW

**Comment by uilianries** [comment]
@lasote KB-H023 is available on the Wiki: https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#KB-H023

**Comment by uilianries** [comment]
> Write the KB explanation that will go on the wiki in the PR description

@lasote, Done!

File: hooks/conan-center.py [hunk 1]
```diff
@@ -25,7 +25,8 @@
              "KB-H018": "LIBTOOL FILES PRESENCE",
              "KB-H019": "CMAKE FILE NOT IN BUILD FOLDERS",
              "KB-H020": "PC-FILES",
-             "KB-H021": "MS RUNTIME FILES"}
+             "KB-H021": "MS RUNTIME FILES",
+             "KB-H023": "EXPORT RECIPE"}
```
Comment (lasote) [inline hooks/conan-center.py]: `EXPORT LICENSE`?

Comment (uilianries) [reply hooks/conan-center.py]: :joy_cat:

Comment (uilianries) [reply hooks/conan-center.py]: Done!

**Comment by danimtb** [comment]
I have added the entry in the knowledge base https://github.com/conan-io/conan-center-index/wiki/Error-Knowledge-Base#kb-h023-export-license

**Pull request merged**